src/sc.c

```diff
@@ -1293,7 +1293,9 @@ sc_init (sc_MPI_Comm mpicomm,
          int catch_signals, int print_backtrace,
          sc_log_handler_t log_handler, int log_threshold)
 {
-  int                 w;
+#if 0
+  const int           w = 24;
+#endif
```

Comment:
The `#if 0` removes the `const int` definition. Wouldn't that lead to a compiler error in lines 1366ff where you use `w` in a macro?

Reply:
True, if we reset one of the two #if 0 to 1 and not the other, this will not work. This is a bit of lazy shortcut expecting that the two definitions apply to the same context. Maybe I should name the define. Let me do that.

Reply:
Ok I just killed it.